### PR TITLE
Add boolean to enable/disable TPC QA plots + add centralityVsNch QA plot

### PR DESCRIPTION
**Pull request by romainschotter**
- Add boolean to enable/disable TPC QA plots
- Add centralityVsNch QA plot in real and MC data processing functions

### CODE REVIEW

File: PWGLF/Tasks/Strangeness/derivedlambdakzeroanalysis.cxx [hunk 1]
```diff
@@ -459,6 +467,22 @@ struct derivedlambdakzeroanalysis {
     if (fabs(v0.negTOFDeltaTK0Pi()) < maxDeltaTimePion)
       bitset(bitMap, selTOFDeltaTNegativePionK0Short);
 
+    // TOF PID
+    // Positive track
+    if (fabs(v0.posTOFDeltaTLaPr()) < maxDeltaTimeProton)
```
Comment (ddobrigk) [inline PWGLF/Tasks/Strangeness/derivedlambdakzeroanalysis.cxx:472]: Hi @romainschotter, is this not equivalent to the preceding block? Or maybe I am missing something?... sorry if so!

File: PWGLF/Tasks/Strangeness/derivedlambdakzeroanalysis.cxx [hunk 2]
```diff
@@ -281,56 +283,62 @@ struct derivedlambdakzeroanalysis {
...
-        histos.add("AntiLambda/h3dPosNsigmaTPCvsTrackPtot", "h3dPosNsigmaTPCvsTrackPtot", kTH3F, {axisCentrality, axisPtCoarse, axisNsigmaTPC});
-        histos.add("AntiLambda/h3dNegNsigmaTPCvsTrackPtot", "h3dNegNsigmaTPCvsTrackPtot", kTH3F, {axisCentrality, axisPtCoarse, axisNsigmaTPC});
-        histos.add("AntiLambda/h3dPosTPCsignalVsTrackPtot", "h3dPosTPCsignalVsTrackPtot", kTH3F, {axisCentrality, axisPtCoarse, axisTPCsignal});
-        histos.add("AntiLambda/h3dNegTPCsignalVsTrackPtot", "h3dNegTPCsignalVsTrackPtot", kTH3F, {axisCentrality, axisPtCoarse, axisTPCsignal});
-        histos.add("AntiLambda/h3dPosNsigmaTPCvsTrackPt", "h3dPosNsigmaTPCvsTrackPt", kTH3F, {axisCentrality, axisPtCoarse, axisNsigmaTPC});
-        histos.add("AntiLambda/h3dNegNsigmaTPCvsTrackPt", "h3dNegNsigmaTPCvsTrackPt", kTH3F, {axisCentrality, axisPtCoarse, axisNsigmaTPC});
-        histos.add("AntiLambda/h3dPosTPCsignalVsTrackPt", "h3dPosTPCsignalVsTrackPt", kTH3F, {axisCentrality, axisPtCoarse, axisTPCsignal});
-        histos.add("AntiLambda/h3dNegTPCsignalVsTrackPt", "h3dNegTPCsignalVsTrackPt", kTH3F, {axisCentrality, axisPtCoarse, axisTPCsignal});
+        if (doTPCQA) {
```
Comment (ddobrigk) [inline PWGLF/Tasks/Strangeness/derivedlambdakzeroanalysis.cxx:328]: You should add an `if(doTPCQA)` statement also to the filling of the histograms ([line 610](https://github.com/AliceO2Group/O2Physics/blob/ee7fa8994220556627089e54739a1c38ec4d8ba4/PWGLF/Tasks/Strangeness/derivedlambdakzeroanalysis.cxx#L610) and others) or the task will crash due to histograms not found... can you please adjust? Thanks!

**Review by njacazio — APPROVED** [review]

**Comment by njacazio** [comment]
I will wait for @romainschotter to comment on the status of the changes before merging

**Comment by romainschotter** [comment]
@njacazio you can go ahead and start merging. I have fixed the issues raised by David. Thank you very much!

**Pull request merged**